### PR TITLE
🔖 chore: Bump version to 0.8.1

### PR DESCRIPTION
**Pull request by MfCrizz**
Bump version in Helm Chart and in values.yaml to 0.8.1 for next release.

### CODE REVIEW

File: chart/values.yaml [hunk 1]
```diff
@@ -48,7 +48,7 @@ maliciousLoadGenerator:
     container:
       image:
         repository: ghcr.io/dynatrace-oss/unguard/unguard-malicious-load-generator
-        tag: 0.8.0
+        tag: 0.8.1
```
Comment (W3D3) [inline chart/values.yaml:51]: [NIT] Could we extract this into a variable so for future changes we only change it on a single point in the `values.yaml`?

**Review by W3D3 — APPROVED** [review]

File: chart/values.yaml (hunk 1 above)
Comment (MfCrizz) [reply chart/values.yaml:51]: Yeah could be extracted, but then it is not possible to have different versions of each service anymore.

Comment (MfCrizz) [reply chart/values.yaml:51]: Also it is not really necessary to change all image tags every time. I just did it to have a clean start for the helm chart.
The image tags in the values.yaml get overwritten anyway by the release action.

Comment (W3D3) [reply chart/values.yaml:51]: But I don't think that's a requirement, we already increment every service for every version bump, so they have to be in sync anyways.

**Pull request merged**